### PR TITLE
Getifaddrs

### PR DESCRIPTION
**Pull request by rennergade**
Companion to libc/rustposix PR's. Simply transfers buffer from rustposix to libc.

### CODE REVIEW

**Review by youngfalcon — APPROVED** [review]
Looks good to me.

File: src/trusted/service_runtime/nacl_syscall_common.c [hunk 1]
```diff
@@ -5114,6 +5114,30 @@ int32_t NaClSysGethostname(struct NaClAppThread *natp, char *name, size_t len) {
...
+  struct NaClApp *nap = natp->nap;
+  
+  NaClLog(2, "Cage %d Entered NaClSysGetifaddrs(0x%08"NACL_PRIxPTR", "
+          "0x%08"NACL_PRIxPTR", "
+          "%d)\n",
+          nap->cage_id, (uintptr_t) natp, (uintptr_t) buf, len);
+  
+  sysaddr = NaClUserToSysAddrRangeProt(nap, (uintptr_t) buf, len, NACL_ABI_PROT_WRITE);
+  if (kNaClBadAddress == sysaddr) {
+    NaClLog(2, "NaClSysGethostname could not translate buffer address, returning %d\n", -NACL_ABI_EFAULT);
```
Comment (kapkic) [inline src/trusted/service_runtime/nacl_syscall_common.c]: Shouldn't this be NaClSysGetifaddrs?

**Review by kapkic — COMMENTED** [review]
Looks good.

**Review by justyoyo123 — APPROVED** [review]
Looks good to me.

**Pull request merged**